server/index.js

```diff
@@ -0,0 +1,25 @@
+require('dotenv').config()
+const express = require('express');
+const path = require('path')
+const axios = require('axios');
+const cors = require('cors');
+const morgan = require('morgan');
+
+const app = express();
+
+// middleware
+
+// logging and parsing
+app.use(cors())
+app.use(morgan('dev'))
+app.use(express.json());
+
+// serve up client files
+app.use(express.static(path.join(__dirname, "../client/dist")));
+
+// routes/controllers will utilize client requests and communicate with the API here.
+
+const port = 3000;
```

Comment:
I think we might want to set up port like you mentioned in example.env, if port is not defined in the environment then let it be 3000, instead of just making it 3000

Reply:
I had originally set it up as `const PORT = process.env.PORT || 3000` but I wasn't actually sure if we would need the env file, will we have different environments? So I just set the port to be 3000 and left the example.env. But I can def change it back, I suppose it doesn't hurt?